_browser/blockstack_storage.md

```diff
@@ -680,48 +612,46 @@ process URL paths that contain the `.` (dot) character for example,
                       : 'Nameless Person' }</span>
                   </h1>
                   <span>{username}</span>
-                  {this.isLocal() &&
+                  {isLocal() &&
                     <span>
                       &nbsp;|&nbsp;
-                      <a onClick={ handleSignOut.bind(this) }>(Logout)</a>
+                      <a onClick={handleSignOut}>(Logout)</a>
                     </span>
                   }
                 </div>
               </div>
             </div>
-            {this.isLocal() &&
+            {isLocal() &&
               <div className="new-status">
                 <div className="col-md-12">
                   <textarea className="input-status"
-                    value={this.state.newStatus}
-                    onChange={e => this.handleNewStatusChange(e)}
+                    value={newStatus}
+                    onChange={handleNewStatus}
                     placeholder="What's on your mind?"
                   />
                 </div>
                 <div className="col-md-12 text-right">
                   <button
                     className="btn btn-primary btn-lg"
-                    onClick={e => this.handleNewStatusSubmit(e)}
+                    onClick={handleNewStatusSubmit}
                   >
                     Submit
                   </button>
                 </div>
               </div>
             }
             <div className="col-md-12 statuses">
-            {this.state.isLoading && <span>Loading...</span>}
-            {this.state.statuses.map((status) => (
+              {isLoading && <span>Loading...</span>}
+              {statuses.map((status) => (
                 <div className="status" key={status.id}>
                   {status.text}
                 </div>
-                )
-            )}
+              ))}
             </div>
           </div>
         </div>
-      </div> : null
+      </div>
     );
-    }
     ```
 
     This checks to ensure that users are viewing their own profile, by wrapping the **Logout** button and inputs with the `{isLocal() && ...}` condition.
```

Comment:
At the end it says `Restart the application so that the disabling of the . (dot) rule takes effect.`... what is the dot rule?

Reply:
I have literally no idea.